hazelcast-client/src/main/java/com/hazelcast/client/config/XmlClientConfigBuilder.java

```diff
@@ -92,6 +99,37 @@ public XmlClientConfigBuilder() {
         this.in = locator.getIn();
     }
 
+    @Override
+    protected Document parse(InputStream inputStream) throws Exception {
+
+        final DocumentBuilder builder = DocumentBuilderFactory.newInstance().newDocumentBuilder();
+        try {
+            return builder.parse(inputStream);
+        } catch (final Exception e) {
+            String msg = "Failed to parse Config Stream"
+                    + "\nException: " + e.getMessage()
```

Comment:
this will produce funky output on windows. you should use environment neutral approach like
```java
System.getProperty("line.separator");
```
perhaps we already have a util method for this?